llm_on_ray/inference/hpu_deepspeed_predictor.py

```diff
@@ -0,0 +1,313 @@
+#
+# Copyright 2023 The LLM-on-Ray Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ===========================================================================
+#
+# This file is adapted from
+# https://github.com/HabanaAI/optimum-habana-fork/blob/aa64051ae92d8940a49322b29d1dfa933b2d3250/examples/text-generation/utils.py
+# Copyright 2023 Huggingface
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import tempfile
+import ray
+from inference.torch_dist import (
+    TorchDistributedWorker,
+    init_torch_dist_process_group,
+)
+from ray.util.scheduling_strategies import PlacementGroupSchedulingStrategy
+from ray.util.placement_group import (
+    placement_group,
+)
+import torch
+from optimum.habana.checkpoint_utils import (
+    get_ds_injection_policy,
+    model_on_meta,
+    write_checkpoints_json,
+)
+from predictor import Predictor
+from inference.inference_config import (
```

Comment:
Need to import from llm_on_ray.xxx like below. can you please update all of others in your PR?
```
from llm_on_ray.inference.inference_config import InferenceConfig, GenerateResult
```

Reply:
This PR is now messed up with the rebase... Files are moved. I'm trying to recover